### ISSUE TITLE
I think we need some automatic sanity check for new entries

### ISSUE THREAD

**Issue by marco-2023** [theochem/AtomDBdata#10]
I think we can (and probably should) add some (automated) sanity checks for new entries.

**Comment by msricher** [comment]
Yes, I think so. It can go in the compile function, in between when the constructed Species instance is returned from `{dataset}/run()` and when it is `dump`ed.
